### DISCUSSION TITLE
Email not getting compiled

### DISCUSSION THREAD

**Discussion by Rohith2208** [tabler/tabler#1789] [Q&A]
### Browser

chrome

### OS

windows 10

### Screen size

1330x572

### Describe the bug

when i try to compile the email template using gulp i got the following error 
### [14:21:43] Using gulpfile ~\Downloads\tabler templates\gulpfile.js
[14:21:43] Starting 'compile'...
[14:21:43] 'compile' errored after 47 ms
[14:21:43] Error in plugin 'gulp-premailer'
You need to have Ruby and Premailer installed and in your PATH for this task to work.

i tried installing ruby and premailer using npm

### How to reproduce

2. Click on '...'
3. Scroll down to '...'
...

**Comment by aussitot** [comment]
Same problem for me

**Comment by chunkydotdev** [comment]
Why is the issue closed?
